prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -56,7 +56,7 @@ import kotlinx.serialization.json.Json
 import org.json.JSONObject
 import java.io.InputStream
 import java.security.KeyStore
-import java.util.Base64
+import java.util.*
```

Comment:
Can we roll this back? Seems like we generally avoid * imports.

Reply:
good catch for the trickle in star import

Reply:
Looks like this is still using the `*` import